typedoc.js

```diff
@@ -0,0 +1,133 @@
+// const git = require('git-rev-sync');
+
+const packages = [
+  'application-extension',
```

Comment:
This should also be generated.

Reply:
So far as I can tell, `inputFiles` always gets ignored, so the `packages` array here probably isn't needed. I'm still experimenting

Reply:
I removed that array entirely